encoding/protobuf/doc.go

```diff
@@ -54,11 +54,60 @@
 // are created for every RPC method: one that will handle the standard Protobuf
 // binary encoding, and one that will handle the JSON encoding.
 //
+// If coupled with an HTTP Inbound, Protobuf procedures can be called using
+// curl. Given the following Protobuf definition:
+//
+//   syntax = "proto3;
+//
+//   package foo.bar;
+//
+//   message EchoRequest {
+//     string value = 1;
+//   }
+//
+//   message EchoResponse {
+//     string value = 1;
+//   }
+//
+//   service Baz {
+//     rpc Echo(EchoRequest) returns (EchoResponse) {}
+//   }
+//
+// And the following configuation:
+//
+//   service:
+//     name: hello
+//   yarpc:
+//     inbounds:
+//       http:
+//          address: ":8080"
+//
+// If running locally, one could make the following call:
+//
+//   curl \
+//     http://0.0.0.0:8080 \
+//     -H 'context-ttl-ms: 2000' \
+//     -H 'rpc-caller: hello-client' \
+//     -H 'rpc-service: hello' \
+//     -H 'rpc-encoding: json' \
+//     -H 'rpc-procedure: foo.bar.Baz::Echo' \
+//     -d '{"value":"sample"}'
+//
+// Where context-ttl-ms is the timeout, rpc-caller is anything, rpc-service is the name
```

Comment:
minor nits:

- "context-ttl-ms is the timeout in milliseconds"
- Can we make the example `"rpc-caller: curl-$(whoami)"` so that users default to providing their name in ad-hoc requests? Then the doc for it an be, `rpc-caller` is the name of the entity making the request.
- "rpc-procedure is the name of the protobuf method being called in the form proto_package.proto_service::proto_method"

Reply:
Fixed